5-update-&-rollback.md

```diff
@@ -38,7 +38,7 @@ git commit -m "Added buggy alert"
 Build the container and deploy the latest revision:
 ```bash
 nscale system compile sudc development
-nscale container build sudc web
+nscale container build sudc web development
 nscale revision deploy sudc latest development
 ```
```

Comment:
Should the command be `dev` over `development` here? it seems the docs have `dev` in some places, `development in others`. Unless there is a reason we should perhaps stick to one convention for docs? @darragh-hayes can you check this? @mcollina can you confirm?